ghedesigner/enums.py

```diff
@@ -2,10 +2,10 @@
 
 
 class BHPipeType(Enum):
-    COAXIAL = auto()
-    DOUBLEUTUBEPARALLEL = auto()
-    DOUBLEUTUBESERIES = auto()
-    SINGLEUTUBE = auto()
+    COAXIAL = "COAXIAL"
+    DOUBLEUTUBEPARALLEL = "DOUBLEUTUBEPARALLEL"
+    DOUBLEUTUBESERIES = "DOUBLEUTUBESERIES"
+    SINGLEUTUBE = "SINGLEUTUBE"
```

Comment:
Why the switch to strings here? You can get the string by `EnumType.ENUMVAL.name`.